maps/CEVEris/_Eris_areas.dm

```diff
@@ -461,6 +461,7 @@
 	name = "\improper Bar"
 	icon_state = "bar"
 	sound_env = LARGE_SOFTFLOOR
+	narrate = "Reflective blue floors give the illusion of neon and glamour. The smell of alcohol is strong on the air. "
```

Comment:
I thought this was just backend support?

Reply:
yes, this was the example narrate i left in by accident. i am going to comment it out and explain it is the example narrate.

Reply:
Leave it in. It's less obtrusive than originally envisioned.

May as well add some for certain areas on the ship.